### DISCUSSION TITLE
Feature Request: Adding text non-rotation option to GenPollText for vertical bars.

### DISCUSSION THREAD

**Discussion by Sinaaaa** [qtile/qtile#5137] [Ideas]
![screenshot_2024122613:2704](https://github.com/user-attachments/assets/a87bdd45-58e6-42e3-be50-7e149ed92ff5)
Below and above the vertical clock widget all these are GenPollText widgets. It would be nice if GenPollText had the option to not rotate the text if the bar is rotated.

**Comment by elParaguayo** [comment]
That's easy enough to add but the text would need to be short!

**Reply by Sinaaaa** [reply]
As long as 4 characters long is allowed I'm good :P `100%` is probably the longest thing I'll ever display like this.

**Reply by elParaguayo** [reply]
You'll just be limited by the width of the bar and your font size.
